api/test/sections/sections-crud.spec.ts

```diff
@@ -21,46 +20,45 @@ describe('Page Sections API', () => {
     await dataSourceManager.loadMockData();
 
     const entityMocks = testManager.mocks();
-    const sections = [
-      await entityMocks.createSection({
-        slug: 'section-1',
-        name: 'Section 1',
-        description: ':)',
-        order: 1,
-        baseWidgets: [
-          await entityMocks.createBaseWidget({
-            sectionOrder: 1,
-            indicator: 'Total countries',
-          }),
-          await entityMocks.createBaseWidget({
-            sectionOrder: 2,
-            indicator: 'Non-existent indicator',
-          }),
-          await entityMocks.createBaseWidget({
-            sectionOrder: 3,
-            indicator: 'Organisation by sector',
-          }),
-        ],
-      }),
-      await entityMocks.createSection({
-        slug: 'section-2',
-        name: 'Section 2',
-        description: ':)',
-        order: 2,
-        baseWidgets: [
-          await entityMocks.createBaseWidget({
-            sectionOrder: 1,
-            indicator: 'Total surveys',
-          }),
-        ],
-      }),
-    ];
+    await entityMocks.createSection({
+      slug: 'section-1',
+      name: 'Section 1',
+      description: ':)',
+      order: 1,
+      baseWidgets: [
+        await entityMocks.createBaseWidget({
+          sectionOrder: 1,
+          indicator: 'Total countries',
+        }),
+        await entityMocks.createBaseWidget({
+          sectionOrder: 2,
+          indicator: 'Non-existent indicator',
+        }),
+        await entityMocks.createBaseWidget({
+          sectionOrder: 3,
+          indicator: 'Organisation by sector',
+        }),
+      ],
+    });
+
+    await entityMocks.createSection({
+      slug: 'section-2',
+      name: 'Section 2',
+      description: ':)',
+      order: 2,
+      baseWidgets: [
+        await entityMocks.createBaseWidget({
+          sectionOrder: 1,
+          indicator: 'Total surveys',
+        }),
+      ],
+    });
 
     // When
     const res = await testManager
       .request()
       .get(
-        '/sections?filters[0][name]=eu-member-state&filters[0][operator]==&filters[0][value]=Spain,France',
+        '/sections?filters[0][name]=eu-member-state&filters[0][operator]==&filters[0][values]=Spain&filters[0][values]=France',
```

Comment:
```suggestion
        '/sections?filters[0][name]=eu-member-state&filters[0][operator]==&filters[0][values]=Spain,France'
```

Should'nt work the same way?

Reply:
Do you think this will cause url length issues? The URL length will significantly grow if we take this approach. Are there any risks/limitations? How many filters could be applied at the same time?

Reply:
If we are not going to have any problems, I find this the best approach.

Reply:
@alepefe-vizz Yes, I agree with that. If we want to keep the comma separated format for values we have to modify `generateEntityQuerySchema` so it will return the correct query string